tests/std/test.lst

```diff
@@ -262,6 +262,7 @@ tests\P0896R4_ranges_alg_generate_n
 tests\P0896R4_ranges_alg_heap
 tests\P0896R4_ranges_alg_is_permutation
 tests\P0896R4_ranges_alg_is_sorted
+tests\P0896R4_ranges_alg_lexi

```

Comment:
I would like to see the algorithm's name fully spelled out, for ease of searching. We shouldn't be close to the arbitrary path length limit that we recently imposed.